test/Interop/Cxx/class/access/private-fileid-irgen.swift

```diff
@@ -0,0 +1,112 @@
+//--- blessed.swift
+// RUN: split-file %s %t
```

Comment:
This is the only file in this test case right? We should skip `split-file` and just compile `%s` directly.

Reply:
This test case relies on the Swift source file being called `blessed.swift`. I'd like to keep it this way to ensure independence between the file name of the test case vs the file name used in the test.

Reply:
Correct me if I missed something, but I don't see anything referring to `blessed.swift` other than the RUN line. I don't see how replacing `%t/blessed.swift` with `%s` doesn't keep the same file name independence, especially since the first RUN line already uses `%s`: `split-file %s %t`. Meanwhile using `split-file` has the downside of _having_ to re-run the lit-test each time you change the test file, instead of just running the command extracted from the RUN line directly, since doing that would still compile the outdated `blessed.swift`. I don't think it's worth using `split-files` in a single file test case like this.

Reply:
`blessed.swift` appears in the header file this test uses. Multiple tests use that header file.

Reply:
That makes sense then!